kong/cmd/utils/prefix_handler.lua

```diff
@@ -248,6 +249,19 @@ local function compile_conf(kong_config, conf_template, template_env_inject)
     }
   }
 
+  local kong_proxy_access_log = kong_config.proxy_access_log
+  if kong_proxy_access_log ~= "off" then
+    compile_env.proxy_access_log_enabled = true
+  end
+  if kong_proxy_access_log then
+    if (not lfs.attributes(kong_proxy_access_log)) and
+       (not lfs.attributes(kong_config.prefix .. "/" .. kong_proxy_access_log))
+    then
+      log.debug(table.concat({"'proxy_access_log' is a custom value:", kong_proxy_access_log}, " "))
+      compile_env.proxy_access_log_custom = true
```

Comment:
If I understand this correctly it is trying to detect whether the `access_log` was configured using some custom format like: `access_log /tmp/access.log custom_format;`

It seems this code will return true when passing `KONG_PROXY_ACCESS_LOG=$(pwd)/some_file.log` if the target file (`some_file.log`) didn't already exist, should we rather detect if a _format_ exists after the path and only then consider it custom?

Reply:
@samugi nice catch.

To determine if a *format* exist or not, we then use *space* to split the `access_log`.

Previously, I worry if the `some_file.log` may contain white spaces, but seems it is the user's responsibility to set a correct filename (with quotes).

Let me update.

Reply:
that sounds reasonable to me

Reply:
@samugi updated.